src/Metadata/Source/SqliteMetadata.php

```diff
@@ -375,7 +420,10 @@ protected function getIdentifierChainRegularExpression()
         return $re;
     }
 
-    protected function getIdentifierListRegularExpression()
+    /**
+     * @return string|null
```

Comment:
we can remove comments like that as it does not provide any additional information - we have it clear in method signature

Reply:
@webimpress 
I have already given him the link to the coding-standard 2.0 on Slack.

Reply:
Comments with no valuable information have been removed.